server/models/likedislike.js

```diff
@@ -0,0 +1,16 @@
+module.exports = (sequelize, DataTypes) => {
+  const LikeDislike = sequelize.define('LikeDislike', {
+    userId: DataTypes.UUID,
+    articleId: DataTypes.UUID,
+    likeCount: DataTypes.INTEGER,
```

Comment:
Kindly implement the required changes as requested above for the both `dislikeCount` and `likeCount` fields.